extension/chrome/elements/compose-modules/compose-recipients-module.ts

```diff
@@ -383,6 +383,11 @@ export class ComposeRecipientsModule extends ViewModule<ComposeView> {
     this.setEmailsPreview();
   };
 
+  private queryIfGoogleSearchEnabled = async () => {
+    const scopes = await AcctStore.getScopes(this.view.acctEmail);
+    return scopes.readContacts && scopes.readOtherContacts;
```

Comment:
I'd me more comfortable, if we wrap this call in try/catch and return `undefined` in case of exception. (So we can try and search Google contacts in case of `!== false` and leave
```
      if (!(await this.googleContactsSearchEnabled)) {
        this.addBtnToAllowSearchContactsFromGoogle(input);
      }
```
as it is now.